### PR TITLE
Add debug formatting for LLVM Values.

### PR DESCRIPTION
**Pull request by ptersilie**

### CODE REVIEW

File: ykrt/src/frame/llvmbridge.rs [hunk 1]
```diff
@@ -2,6 +2,8 @@ use llvm_sys::core::*;
 use llvm_sys::prelude::{LLVMBasicBlockRef, LLVMModuleRef, LLVMTypeRef, LLVMValueRef};
 use llvm_sys::target::{LLVMGetModuleDataLayout, LLVMTargetDataRef};
 use llvm_sys::LLVMTypeKind;
+use std::ffi::CStr;
```
Comment (ltratt) [inline ykrt/src/frame/llvmbridge.rs:5]: Prefer `use std::{ffi::CStr, fmt}`

Comment (ptersilie) [reply ykrt/src/frame/llvmbridge.rs]: Fixed in [6e8780d](https://github.com/ykjit/yk/pull/769/commits/6e8780dfe8977b43fcb04c0d4586ced4dd00c93f)

**Comment by ptersilie** [comment]
Ready for rereview.

**Comment by ltratt** [comment]
Please squash.

**Comment by ptersilie** [comment]
Squashed.

**Pull request merged**